site/content/en/docs/concepts/multikueue.md

```diff
@@ -43,6 +43,13 @@ The worker cluster acts like a standalone Kueue cluster.
 The **MultiKueue Admission Check Controller**, running in the manager cluster,
 creates and deletes Workloads and Jobs in the worker clusters as needed.
 
+### Cluster Role Sharing
+
+With Cluster Role Sharing capability, you can consolidate your Kueue infrastructure by using the same manager cluster to run both MultiKueue-managed distributed workloads and regular Kueue workloads executed locally.
+MultiKueue and regular ClusterQueues have independent admission paths and quotas, so workloads dispatched to worker clusters do not compete with local jobs for resources. 
+MultiKueue jobs are routed to worker clusters, while regular jobs stay managed and executed in the manager cluster. 
+
```

Comment:
TBH this paragraph is quite heavy to read, but the message is very simple. What about just:

```suggestion
MultiKueue supports cluster role sharing where the manager cluster can also run regular Jobs.
It does not require any extra configuration except for creating a dedicated ClusterQueue for
running regular workloads which are run locally on the cluster.
```
Or even to put this content as a not in the "manager cluster"